.github/workflows/supabase-branch.yml

```
@@ -0,0 +1,172 @@
+      
+      - name: Setup Doppler
+        run: |
+          doppler setup --no-interactive --token=${{ secrets.DOPPLER_TOKEN }} --project=lpg --config=dev
+      
+      # This step is no longer needed since we use PR number directly
+      # But keeping a simplified version for backwards compatibility
+      - name: Extract branch name
+        id: extract_branch
+        run: |
+          echo "branch_name=pr-${{ github.event.number }}" >> $GITHUB_OUTPUT
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)